drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -183,7 +184,7 @@ static int kinetis_usb_init(void)
 
 	irq_enable(CONFIG_USBD_KINETIS_IRQ);
 
-	SYS_LOG_DBG("");
+	LOG_DBG("");
```

Comment:
I assume this does not print function name, right? I think we need to invent some macro like networking and other subsys are doing to overcome missing function name limitation.

Reply:
Thanks for the pointer. Done.